src/utils/shared.spec.js

```diff
@@ -25,6 +25,27 @@ describe('Shared methods and properties', () => {
     expect(shared.resolveAssetsPath).toEqual(jasmine.any(Function));
   });
 
+  it('should add the providers array to the @ngModule if none is present.', () => {
+    const content = new Buffer(`
+      @NgModule({
+        imports: [
+          StacheModule
+        ],
+        exports: [
+          StacheModule
+        ]
+      });`
+    );
+    const withProviders = shared.addToProviders(content, 'SOME_PROVIDERS');
+    expect(withProviders).toContain(`
+      @NgModule({
+        providers: [
+      /* tslint:disable:trailing-comma */
+      SOME_PROVIDERS,
+      /* tslint:enable:trailing-comma */`
```

Comment:
I would probably consider this feedback non-blocking, but I might suggest adding the closing bracket for providers to the test.  Almost might be worth making sure the string doesn't have more than one instance of providers either.